org.salutatoi.Cagou.json

```diff
@@ -0,0 +1,1463 @@
+{
+    "app-id": "org.salutatoi.Cagou",
+    "default-branch": "0.7.0",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "sat_wrapper",
+    "finish-args": [
+        "--socket=session-bus",
+        "--socket=x11",
+        "--share=ipc",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "python",
+            "config-opts": [
+                "--enable-deterministic-archives",
+                "--disable-static",
+                "--enable-shared",
+                "--with-ensurepip=yes",
+                "--with-system-expat",
+                "--with-system-ffi",
+                "--enable-loadable-sqlite-extensions",
+                "--with-dbmliborder=gdbm",
+                "--enable-unicode=ucs4"
+            ],
+            "post-install": [
+                "chmod 644 /app/lib/libpython2.7.so.1.0"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.python.org/ftp/python/2.7.16/Python-2.7.16.tar.xz",
+                    "sha256": "f222ef602647eecb6853681156d32de4450a2c39f4de93bd5b20235f2e660ed7"
+                }
+            ]
+        },
+        {
+            "name": "libxslt",
```

Comment:
All modules likely need some cleanups to delete development headers (etc.).

Reply:
I'll try to review non python modules to see what I can remove.